.pre-commit-config.yaml

```diff
@@ -11,6 +11,16 @@ repos:
       - id: isort
         name: isort
         entry: isort --profile google
+  - repo: https://github.com/psf/black
+    rev: 24.2.0
+    hooks:
+      - id: black
+        files: (sdk|examples)/.*
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.1.1
+    hooks:
+      - id: flake8
+        files: (sdk|examples)/.*
```

Comment:
@Ygnas Please can create PR to add the `pkg` directory as well ?
We have some Python files also: https://github.com/kubeflow/katib/tree/master/pkg/suggestion/v1beta1, https://github.com/kubeflow/katib/tree/master/pkg/earlystopping/v1beta1/medianstop

Reply:
Good point.